go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -14,7 +15,12 @@ import (
 	"text/template"
 	"time"
 
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	apimachinerytypes "k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/strategicpatch"
 	"k8s.io/klog/v2"
+	"k8s.io/kubernetes/pkg/util/taints"
```

Comment:
We shouldn't vendor in `k8s.io/kubernetes` - can this be copy-pasted?

Reply:
I don't follow, we already have the `k8s.io/kubernetes` vendored.. I am just trying to add this package taints similar to what we do in SDN.